src/scss/_use-cases.scss

```diff
@@ -17,36 +17,48 @@
 ///                     _deprecated: <msg>  Emits <msg> as a warning if referenced, but still works
 ///
 /// @type map
-$o-colors-usecases: map-merge((
+$branded-usecases: (
+	'master': (
+	//	<use case>                <properties>
+		page:                     (background: 'paper'),
+		box:                      (background: 'wheat'),
+		link:                     (text: 'teal'),
+		link-hover:               (text: 'black-70'),
+		link-title:               (text: 'black-80'),
+		link-title-hover:         (text: 'black-70'),
+		tag-link:                 (text: 'claret'),
+		tag-link-hover:           (text: 'claret-30'),
+		opinion-tag-link:         (text: 'oxford'),
+		opinion-tag-link-hover:   (text: 'oxford-30'),
+		title:                    (text: 'black'),
+		body:                     (text: 'black-80'),
+		muted:                    (text: 'black-20'),
+		opinion:                  (background: 'sky'),
+		hero:                     (background: 'wheat'),
+		hero-opinion:             (background: 'oxford'),
+		hero-highlight:           (background: 'claret'),
 
-//	<use case>                <properties>
+		// Section colors
+		section-life-arts:        (all: 'velvet'),
+		section-life-arts-alt:    (all: 'candy'),
+		section-magazine:         (all: 'oxford'),
+		section-magazine-alt:     (all: 'sky'),
+		section-house-home:       (all: 'jade'),
+		section-house-home-alt:   (all: 'wasabi'),
+		section-money:            (all: 'crimson'),
+		section-money-alt:        (all: 'white'),
+	),
+	'internal': (
+	//	<use case>                <properties>
+		page:                     (background: 'white'),
+		box:                      (background: 'internal-slate-5'),
+		link:                     (text: 'teal'),
+		link-hover:               (text: 'internal-slate-15'),
+		link-title:               (text: 'internal-slate-15'),
+		link-title-hover:         (text: 'internal-slate-15'),
+		title:                    (text: 'internal-slate'),
+		body:                     (text: 'internal-slate'),
+	)
+);
 
-	page:                     (background: 'paper'),
-	box:                      (background: 'wheat'),
-	link:                     (text: 'teal'),
-	link-hover:               (text: 'black-70'),
-	link-title:               (text: 'black-80'),
-	link-title-hover:         (text: 'black-70'),
-	tag-link:                 (text: 'claret'),
-	tag-link-hover:           (text: 'claret-30'),
-	opinion-tag-link:         (text: 'oxford'),
-	opinion-tag-link-hover:   (text: 'oxford-30'),
-	title:                    (text: 'black'),
-	body:                     (text: 'black-80'),
-	muted:                    (text: 'black-20'),
-	opinion:                  (background: 'sky'),
-	hero:                     (background: 'wheat'),
-	hero-opinion:             (background: 'oxford'),
-	hero-highlight:           (background: 'claret'),
-
-	// Section colors
-	section-life-arts:        (all: 'velvet'),
```

Comment:
I hope internal products aren't using these use-cases but as they could be this is technically breaking. 

We could keep these in the `internal` brand until the next breaking with a deprecation warning. It works by adding a property `_deprecated` to the usecase. E.g.
Something like
```
section-life-arts:        (all: 'velvet', _deprecated: 'It is not a valid usecase for the "internal" brand. Please contact Origami if you have any questions.'),
```